optimum/exporters/openvino/__main__.py

```diff
@@ -308,21 +316,61 @@ class StoreAttr(object):
 
         GPTQQuantizer.post_init_model = post_init_model
 
-    model = TasksManager.get_model_from_task(
-        task,
-        model_name_or_path,
-        subfolder=subfolder,
-        revision=revision,
-        cache_dir=cache_dir,
-        token=token,
-        local_files_only=local_files_only,
-        force_download=force_download,
-        trust_remote_code=trust_remote_code,
-        framework=framework,
-        device=device,
-        library_name=library_name,
-        **loading_kwargs,
-    )
+    model = None
+    if library_name == "open_clip":
+        if is_open_clip_available():
+            import open_clip
+            model, _ = open_clip.create_model_from_pretrained(f'hf-hub:{model_name_or_path}', cache_dir=cache_dir, force_custom_text=True)
+            # tokenizer = open_clip.get_tokenizer(f'hf-hub:{model_name_or_path}')
+            # preprocessors = [tokenizer, preprocessor]
+
+            if not getattr(model, 'config', None):
+                model_config = None
+                try:
+                    model_config = AutoConfig.from_pretrained(model_name_or_path)
+                except:
+                    pass
+
+                if model_config is None:
+                    try:
+                        config_path = hf_hub_download(repo_id=model_name_or_path, filename="open_clip_config.json", cache_dir=cache_dir)
+                        cfg = {}
+                        with open(config_path, 'r', encoding='utf-8') as f:
+                            cfg = json.load(f)
+                        model_config = CLIPConfig(text_config_dict=cfg['model_cfg']['vision_cfg'], vision_config_dict=cfg['model_cfg']['text_cfg'])
+                    except:
+                        pass
+
+                if model_config is None:
+                    raise ImportError(
+                        f"Could not load config for model {model_name_or_path}"
+                    )
+
+                setattr(model, 'config', model_config)
+
+            if not getattr(model.config, 'model_type', None):
+                setattr(model.config, 'model_type', 'clip')
+                setattr(model.config, 'export_model_type', 'clip')
+        else:
+            raise ImportError(
+                "Could not import open_clip module, please, install it to download and convert model"
+            )
```

Comment:
can it be moved in separated function? I think in future if we will integrate it in common optimum part, it will simplifyrefactoring

Reply:
moved to OpenClipForZeroShotImageClassification